Cargo.toml

```diff
@@ -237,12 +237,13 @@ opt-level = 1
 # Dependencies that should be kept in sync through the whole workspace
 [workspace.dependencies]
 anyhow = "1.0.71"
-arrow-array = "50.0.0"
+arrow = "52"
+arrow-array = "52"
 arc-swap = { version = "1.5.1", features = ["serde"] }
 assert_cmd = "2.0.6"
-async-graphql = "6.0.7"
-async-graphql-axum = "6.0.7"
-async-graphql-value = "6.0.7"
+async-graphql = "=7.0.1"
```

Comment:
Latest async-graphql is 7.0.7. An specific reason why we need 7.0.1?

Reply:
because they released a semver breaking change between 7.0.1 and 7.0.2 that i didn't want to deal with right now :/

Reply:
I see, well better than before. Thanks!